looprpc/server.proto

```diff
@@ -177,3 +206,76 @@ message ServerLoopOutPushPreimageRequest {
 
 message ServerLoopOutPushPreimageResponse {
 }
+
+message SubscribeUpdatesRequest {
```

Comment:
Shouldn't these rather be added in the subscribe PR since they're not used?

Reply:
+1